Microsoft.Toolkit.Uwp.UI.Controls.Graph/PlannerTaskList/PlannerTaskList.cs

```diff
@@ -0,0 +1,346 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Threading.Tasks;
+using Microsoft.Graph;
+using Microsoft.Toolkit.Services.MicrosoftGraph;
+using Windows.UI.Popups;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Graph
+{
+    /// <summary>
+    /// The PlannerTaskList Control displays a simple list of Planner tasks.
+    /// </summary>
+    [TemplatePart(Name = ControlTasks, Type = typeof(ListView))]
+    [TemplatePart(Name = ControlInput, Type = typeof(TextBox))]
+    [TemplatePart(Name = ControlAdd, Type = typeof(Button))]
+    [TemplateVisualState(GroupName = MobileVisualStateGroup, Name = MobileVisualState)]
+    public partial class PlannerTaskList : Control
```

Comment:
This should probably be an ItemsControl, or a ListViewBase control since it will be showing a list of items

Reply:
@skendrot The `ItemsControl` controls contains some properties which doesn't apply to our scenario, for example, we don't want to expose the `Items` `ItemsSource` properties as they are expected to be loaded inside our control to avoid troubles.

Reply:
Why does it matter if Items or ItemsSource is exposed? Same as exposing Tasks.

Reply:
@skendrot Exposing the Items and ItemsSource properties mean the user will have the ability to add/remove/update items directly, which isn't a good idea in our scenario and might cause troubles (e.g. adding a task doesn't belong to the selected plan), we'd prefer to keep that as blackbox to users.